src/main/java/org/apache/sysml/parser/ParameterizedBuiltinFunctionExpression.java

```diff
@@ -307,7 +314,65 @@ public void validateExpression(MultiAssignmentStatement stmt, HashMap<String, Da
 				raiseValidateError("Unsupported parameterized function "+ getOpCode(), false, LanguageErrorCodes.INVALID_PARAMETERS);
 		}
 	}
-	
+
+
+	private void validateParamserv(DataIdentifier output, boolean conditional) {
+		String fname = getOpCode().name();
+		// validate the first five arguments
+		if (getVarParams().size() < 1) {
+			raiseValidateError("Should provide more arguments for function " + fname, conditional, LanguageErrorCodes.INVALID_PARAMETERS);
```

Comment:
You can make this error unconditional (pass false instead of conditional) as otherwise it would turn into a warning if it is not guaranteed to be executed e.g., in an if branch.